bundles/org.openhab.binding.hue/src/main/resources/OH-INF/thing/Clip2Thing.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="hue" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- API v2 device thing -->
+	<thing-type id="device">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge-api2"/>
+		</supported-bridge-type-refs>
+
+		<label>Hue Device</label>
+		<description>A Hue API v2 device with channels depending on its actual capabilities.</description>
+
+		<channels>
+			<channel id="color" typeId="system.color"/>
+			<channel id="color-temperature" typeId="system.color-temperature"/>
+			<channel id="brightness" typeId="system.brightness"/>
+			<channel id="switch" typeId="system.power"/>
+			<channel id="alert" typeId="alert-v2">
+				<description>Activate the alert for the light.</description>
+			</channel>
+			<channel id="effect" typeId="effect-v2">
+				<description>Activate the effect for the light.</description>
+			</channel>
+			<channel id="button-last-event" typeId="button-last-event"/>
+			<channel id="rotary-steps" typeId="rotary-steps"/>
+			<channel id="motion" typeId="system.motion"/>
+			<channel id="motion-enabled" typeId="sensor-enabled">
+				<description>Motion sensor enabled.</description>
+			</channel>
+			<channel id="light-level" typeId="illuminance">
+				<description>Current light level.</description>
+			</channel>
+			<channel id="light-level-enabled" typeId="sensor-enabled">
+				<description>Light level sensor enabled.</description>
+			</channel>
+			<channel id="temperature" typeId="system.indoor-temperature"/>
```

Comment:
I didn't check, but if this is also used for temperature for outdoor motion sensor, then probably label/description should be modified.

Reply:
Ok. Done.